labgrid/remote/client.py

```diff
@@ -1527,6 +1528,10 @@ def main():
                            help="skip n 512-sized blocks at start of input")
     subparser.add_argument('--seek', type=int, default=0,
                            help="skip n 512-sized blocks at start of output")
+    subparser.add_argument('--mode', dest='write_mode',
+                           choices=[m.value for m in USBStorageWriteMode],
+                           default=USBStorageWriteMode.DD.value,
+                           help="Choose tool for writing images")
```

Comment:
```suggestion
    subparser.add_argument('--mode', dest='write_mode', choices=USBStorageWriteMode,
                           type=USBStorageWriteMode, default=USBStorageWriteMode.DD,
                           help="Choose tool for writing images (default: %(default)s)")
```
Also added default to the help text.